packages/stable/src/api/annotations/types.gen.ts

```diff
@@ -38,7 +38,7 @@ export type AnnotationsAssetRef = {
 export interface AnnotationsBoolean {
   /** The description of a primitive */
   description?: string;
-  type: 'boolean';
```

Comment:
This is strictly speaking a breaking change... Is `'boolean'` actually the correct type here?